swift/TasksUITests/TasksUITests.swift

```diff
@@ -0,0 +1,100 @@
+import XCTest
+
+final class TasksUITests: XCTestCase {
+
+    func testDidFindOnlyGitHubSeededDocument() throws {
+        let app = XCUIApplication()
+        app.launch()
+
+        XCTAssertTrue(app.wait(for: .runningForeground, timeout: 30),
+                      "App should launch successfully")
+
+        // Get seeded GitHub identifiers from environment (set by BrowserStack setEnvVariables)
+        let githubRunId = ProcessInfo.processInfo.environment["GITHUB_RUN_ID"] ?? ""
```

Comment:
Nit: all caps acronyms in Swift.

```suggestion
        let githubRunID = ProcessInfo.processInfo.environment["GITHUB_RUN_ID"] ?? ""
```